src/paperbot/infrastructure/adapters/arxiv_search_adapter.py

```
@@ -0,0 +1,50 @@
+        self._connector = connector or ArxivConnector()
+
+    @property
+    def source_name(self) -> str:
+        return "arxiv"
+
+    async def search(
+        self,
+        query: str,
+        *,
+        max_results: int = 30,
+        year_from: Optional[int] = None,
```

Ruff (0.15.0)

[warning] Unused method argument: `year_from`

(ARG002)